DatadogSDKBridge.podspec.src

```diff
@@ -14,6 +14,7 @@ Pod::Spec.new do |s|
   s.homepage     = "https://www.datadoghq.com"
   s.social_media_url   = "https://twitter.com/datadoghq"
 
+  s.swift_versions        = ['5.1']
```

Comment:
currently Xcode seems to support `4` , `4.2` and `5` as Swift version.
it shouldn't make any difference but for the sake of clarity, can we set it to `[5]`?

Reply:
I took it from iOS repo https://github.com/DataDog/dd-sdk-ios/blob/696b394bccbe85711fb353de1f5c54f1a3868a11/DatadogSDK.podspec#L17. Should it be updated there as well then?

Reply:
hmm okay let's keep it `5.1` then 👍 cocoapods should map both of them to `5` anyway